src/Twig/FrontendMenuExtension.php

```diff
@@ -50,4 +56,11 @@ public function renderMenu(Environment $twig, ?string $name = null, string $temp
 
         return $twig->render($template, $context);
     }
+
+    public function isCurrent($item): bool
+    {
+        $uri = $item['uri'] ?? '';
+
+        return $uri === $this->request->getUri();
```

Comment:
I think it should say `return $uri === $this->request->getRequestUri();` here, as `getUri()` returns the absolute URI and `getRequestUri()` returns the absolute path.

Reply:
Yes, you're absolutely right! Thanks for spotting this! :-)